### DISCUSSION TITLE
Nach XML.API2.0-Rollout keine Virtuelle Geräte (Tasten HmIP-RCV-50 HmIP-RCV und HM-RCV-50 BisCos-RF) in state.list zu finden

### DISCUSSION THREAD

**Discussion by HeiNieKu** [OpenCCU/OpenCCU#2529] [🇩🇪 Fragen&Antworten] [answered]
Hi zusammen, 
ich habe ein Problem seit XML-API2.0 ausgerollt wurde. 
Die neue Sicherheit → super.
Probleme mit der TinyMatic-App sind inzwischen besten gelöst.
Danke
Ich dachte das Problem, mit dem ich kämpfe erledigt sich beim nächsten Rollout von XML-API
auch. Inzwischen sind wir bei XML-API2.3, mein Problem steht noch immer. Vielleicht bin aber
auch ich das Problem oder nur zu ….
Seit dem XML.API2.0 -Rollout finde ich in der statelist.cgi keine Daten mehr zu den Virtuellen Tasten
(HmIP-RCV-50 HmIP-RCV und HM-RCV-50 BisCos-RF). Die Informationen zu den Tasten sind da, ich kann diese über Scripte ansprechen und mit diesen arbeiten. Lästig ist aber der Wechsel zur XML-API1.22 um Informationen zu holen und dann wieder zurück zur neuen Version 2.3.
Ich kratze mit meinen Wissen nur an der Oberfläche der Materie und hoffe, dass da jemand auf meine Frage antwortet (etwa):
Ja dann schau halt da rein, da findest du sie nach wie vor.----> super
oder
Diese Daten braucht niemand, deshalb wurden diese aus der state.list entfernt ---> schade 
oder
Ups, sie werden wieder kommen ---> happy

In Hoffnung,
Danke im Voraus

**Comment by jens-maus** [comment, marked as answer]
Das gehört hier nicht her. Das ist das RaspberryMatic Diskussionsforum und nicht das XML-API Projekt. Daher: Das gehört im XML-API Projekt entsprechend thematisiert. Siehe https://github.com/homematic-community/XML-API
